com.unity.netcode.gameobjects/Tests/Runtime/HiddenVariableTests.cs

```diff
@@ -211,7 +212,7 @@ public IEnumerator HiddenVariableTest()
             Debug.Log("Values changed");
 
             // ==== Show our object again to this client
-            HiddenVariableObject.ExpectedSize = 3;
+            HiddenVariableObject.ExpectedSize = 2;
```

Comment:
huh?

Reply:
what's the motivation behind 2 -> 3, magic numbers? :P

Reply:
Commented throughout.